src/RemoteInboxNotifications/EvaluateAndGetStatus.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Evaluates the spec and returns a status.
+ *
+ * @package WooCommerce Admin/Classes
+ */
+
+namespace Automattic\WooCommerce\Admin\RemoteInboxNotifications;
+
+defined( 'ABSPATH' ) || exit;
+
+use \Automattic\WooCommerce\Admin\Notes\WC_Admin_Note;
+
+/**
+ * Evaluates the spec and returns a status.
+ */
+class EvaluateAndGetStatus {
```

Comment:
I don't think having this as a class really provides a benefit over a single function like `evaluate_spec_and_get_status( $spec, $current_status, $rule_evaluator )`.

Reply:
Done in 6e616cc7